src/pages/[platform]/build-a-backend/add-aws-services/predictions/identify-text/index.mdx

```diff
@@ -0,0 +1,175 @@
+    props: {
+      meta
+    }
+  };
+}
+
+## Working with the API
+
+Detect text in an input image. Input can be sent directly from the browser or an Amazon S3 key from project bucket.
+
+```javascript
+import { Predictions } from '@aws-amplify/predictions';
+
+const response = await Predictions.identify({
+  text: {
+    source: {
+      file
+    }
+  }
+});
+console.log({ response });
+```
+
+## Identify image stored in Amazon S3
+
+```javascript
+import { Predictions } from '@aws-amplify/predictions';
+
+const response = await Predictions.identify({
+  text: {
+    source: {
+      key: pathToPhoto,
+      level?: 'guest' | 'private' | 'protected', //optional, default is configured on Storage category
+    }
+  }
+})
+console.log({ response });
+```
+
+> The following options are independent of which `source` is specified. For demonstration purposes we will reference a `file` but it can be an S3 Key as well. `Predictions.identify({text : {...}})` can detect unstructured text `PLAIN`, structured text from tables `TABLE` or text from forms `FORM`.
+
+## Identify plain text
+
+For detecting plain text, you can see the whole detected text, the lines detected, the position of each line of text, and each word.
+
+```javascript
+import { Predictions } from '@aws-amplify/predictions';
+
+const response = await Predictions.identify({
+  text: {
+    source: {
+      file
+    },
+    format: 'PLAIN'
+  }
+});
+
+const {
+  text: {
+    fullText, // String
+    lines, // Array of String ordered from top to bottom
+    linesDetailed /* Array of objects that contains
+        text, // String
+        boundingBox: {
+          width, // ratio of overall image width
+          height, // ratio of overall image height
+          left, // left coordinate as a ratio of overall image width
+          top // top coordinate as a ratio of overall image height
+        },
+        polygon // Array of { x, y } coordinates as a ratio of overall image width and height
+        */,
+    words // Array of objects that contains { text, boundingBox, polygon}
+  }
+} = response;
+```
+
+## Identify structured forms
+
+For detecting structured forms (documents, tables, etc.) from an image, `keyValues` will return a string of the entity found in the image as well as metadata such as selected checkboxes or the relative location in the image using a `boundingBox`.
+
+```javascript
+import { Predictions } from '@aws-amplify/predictions';
+
+const response = await Predictions.identify({
+  text: {
+    source: {
+      file
+    },
+    format: 'FORM'
+  }
+});
+
+const {
+  text: {
+    // same as PLAIN +
+    keyValues // Array of { key: string, value: { text: string, selected: boolean}, polygon, boundingBox }
+  }
+} = response;
+```
+
+For example the below image would return `keyValues` with "Test" or "Checked" as a key, and `true` since they are selected. The location of these elements would be returned in the `boundingBox` value.
+
+![Image](/images/IdentifyTable.png)
```

Comment:
Could we get a better alt text for this @offlineprogrammer? I see that this was the alt text for it previously, but could we please improve it so that it is more descriptive?

Here are some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

Reply:
done